wpilibj/src/main/java/edu/wpi/first/wpilibj/PWM.java

```diff
@@ -194,23 +189,23 @@ public double getSpeed() {
   /**
    * Set the PWM value directly to the hardware.
    *
-   * <p>Write a raw value to a PWM channel.
+   * <p>Write a microsecond pulse value to a PWM channel.
    *
-   * @param value Raw PWM value. Range 0 - 255.
+   * @param millisecondPulseTime Microsecond pulse PWM value. Range 0 - 4.096.
    */
-  public void setRaw(int value) {
-    PWMJNI.setPWMRaw(m_handle, (short) value);
+  public void setPulseTimeMicroseconds(int millisecondPulseTime) {
+    PWMJNI.setPulseTimeMicroseconds(m_handle, millisecondPulseTime);
```

Comment:
This parameter is still named `millisecondPulseTime` (and the doc comment says max of 4.096, not 4096).